### PR TITLE
Ctn doc deploy branch

### PR DESCRIPTION
**Pull request by comtechnet**
Test of Merge

### CODE REVIEW

**Review by comtechnet — COMMENTED** [review]
Agreed to the doc clarification

File: README.md [hunk 1]
```diff
@@ -50,14 +50,18 @@ Go to the [Flow Testnet Faucet](https://testnet-faucet-v2.onflow.org/) to create
 
 #### Save your keys
 
-After your account has been created, save the address and private key to the following environment variables:
+After your account has been created, export the following variables to your shell:
```
Comment (10thfloor) [inline README.md:53]:
```suggestion
After your account has been created, export the following environment variables to your shell:
```

**Comment by 10thfloor** [comment]
It's important that these values are exported to your shell, and not kept in an `.env` file, since they are required by the flow-cli and by the bash (curl) commands used to bootstrap the project. 

The project is set up to read these values from the current shell's env. 

Updated this PR with documentation that hopefully clarifies this requirement.

**Comment by sideninja** [comment]
> It's important that these values are exported to your shell, and not kept in an `.env` file, since they are required by the flow-cli and by the bash (curl) commands used to bootstrap the project.
> 
> The project is set up to read these values from the current shell's env.
> 
> Updated this PR with documentation that hopefully clarifies this requirement.

CLI knows how to read .env file with this variable, there has been a PR open for this change on kitty-items. If you think it is better to use exported vars that is fine but you can use .env file too with CLI https://docs.onflow.org/flow-cli/security

**Review by 10thfloor — APPROVED** [review]

**Pull request merged**